### DISCUSSION TITLE
[Support]: calling http://frigate/review with a preselected camera and show reviewed

### DISCUSSION THREAD

**Discussion by lukicsl** [blakeblackshear/frigate#12589] [General Support] [answered]
### Describe the problem you are having

I am processing  the event in habapp. 
The result of this processing are pushover messages which contain the thumbnail for preview as well as the snapshot and the link to the review page (former event page). What I would like is to pass in the url something like ?camera=housedoor&show_reviewed=true, in order to filter the specific camera, as well as set the show reviewed flag. 

### Version

0.14.0 beta 4

### Frigate config file

```yaml
not relevant
```

### Relevant log output

```shell
not relevant
```

### FFprobe output from your camera

```shell
not relevant
```

### Frigate stats

```json
not relevant
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

not relevant

**Comment by NickM-27** [comment, marked as answer]
You can use `?id={review_id}` to load a specific event. There is no way to preselect on the filters.

**Reply by lukicsl** [reply]
great!

have some further questions looking at the  review data:
```
{
  "type": "end",
  "before": {
    "id": "1721754795.575895-ubfvks",
    "camera": "front-garden",
    "start_time": 1721754795.575895,
    "end_time": null,
    "severity": "alert",
    "thumb_path": "/media/frigate/clips/review/thumb-front-garden-1721754795.575895-ubfvks.webp",
    "data": {
      "detections": [
        "1721754794.575612-ngnmh2",
        "1721754822.177298-h830pj"
      ],
      "objects": [
        "person"
      ],
      "sub_labels": [],
      "zones": [
        "eingang"
      ],
      "audio": []
    }
  },
```
I extracted for both detections then snapshot data an they showing me going to the trashcan and comming back:
![image](https://github.com/user-attachments/assets/6d13aaf2-3674-4764-bf77-5a9dde73d36c)

![image](https://github.com/user-attachments/assets/cbd09b8f-49e1-46e2-a4e3-965206377f7f)

the id={review_id } start somewhere in the middle of these snapshots (see time marker).
![image](https://github.com/user-attachments/assets/65da60c9-1fdf-49e2-8c78-96bb22039630)

I rather would like to see the review of the first event, or at least start at the first event (detection).

**Reply by NickM-27** [reply]
Reviews will capture the entire time that activity occurred. If there are no recordings at that point then that's separate

**Reply by lukicsl** [reply]
There is a recording, though it starts about 30-40s late.

**Comment by lukicsl** [comment]
looking at the review overview page i see this preview picture, which start showing the preview video at the right spot when hoovering:
![image](https://github.com/user-attachments/assets/427f5225-1d1f-45d9-9ca1-c4c646f72de7)

Klicking on it the actual video start late at the same spot when calling ?id=review_id

**Reply by NickM-27** [reply]
right, this is unrelated to the review logic, that can only play recordings that exist of course. Most likely recordings are not being saved for both events or something else.

**Reply by lukicsl** [reply]
should I open another thread. The review contains 2 events, the playback of recording start inbetween this two events, the hoover fast forward diplay starts at right time as configured in the config.yml, when clicking on it the playback starts some 30s past the first event.

**Reply by NickM-27** [reply]
the hover and playback are entirely separate, yes you should open a new discussion and fill in the requested information. The clicking / ?id works correctly when recordings exist, so this is working right from the UI perspective
